src/services/formatting/smartIndenter.ts

```diff
@@ -464,9 +471,29 @@ namespace ts.formatting {
                 case SyntaxKind.GetAccessor:
                 case SyntaxKind.SetAccessor:
                     return child !== SyntaxKind.Block;
+                case SyntaxKind.ExportDeclaration:
+                    return child !== SyntaxKind.NamedExports;
                 default:
                     return false;
             }
         }
+
+        function hasBraceShell(node: Node) {

```

Comment:
@SaschaNaz currently `nodeIsCompleted` handles cases when code is incomplete i.e.
```ts
import {$ // hit enter at $, caret on the next line should be indented
```.

I would expect imports and export to follow the same technique that we already use for all other blocks\block like constructions